### PR TITLE
new mobs and changes to some old

### PR DESCRIPTION
**Pull request by Boy0000**

### CODE REVIEW

**Comment by Boy0000** [comment]
some animations and textures are still missing. Hisomukuro dissapears every other frame or has unfit textures. Uibita has incorrect textures ingame but not in blockbench

File: assets/minecraft/models/item/mob/hisomukuro/hisomukuro_swim0.json [hunk 1]
```diff
@@ -0,0 +1,581 @@
+{
```
Comment (0ffz) [inline assets/minecraft/models/item/mob/hisomukuro/hisomukuro_swim0.json:1]: put all these frames in a folder either named `excluded` inside of the mob. Anything inside that folder will not be added to the resourcepack on the server. Also unzip any zip files like `hisomukuro` and do the same for them

File: assets/minecraft/models/item/mob/onitsuchi/onitsuchi.json [hunk 2]
```diff
@@ -1,752 +1 @@
-{
```
Comment (0ffz) [inline assets/minecraft/models/item/mob/onitsuchi/onitsuchi.json:1]: Revert deletion, same for move model

File: assets/minecraft/models/item/mob/sakuranbo/sakuranbo.json [hunk 3]
```diff
@@ -1,264 +1 @@
-{
```
Comment (0ffz) [inline assets/minecraft/models/item/mob/sakuranbo/sakuranbo.json:1]: revert deletion

**Review by 0ffz — CHANGES_REQUESTED** [review]
Make sure all hit models are removed and all deleted mob models are reverted. I didn't comment all of them so make 100% sure they're all rolled back!

**Pull request closed without merging**